content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -329,17 +326,28 @@ PersistentVolume types are implemented as plugins. Kubernetes currently supports
 * [`local`](/docs/concepts/storage/volumes/#local) - local storage devices
   mounted on nodes.
 * [`nfs`](/docs/concepts/storage/volumes/#nfs) - Network File System (NFS) storage
-* `photonPersistentDisk` - Photon controller persistent disk.
-  (This volume type no longer works since the removal of the corresponding
-  cloud provider.)
 * [`portworxVolume`](/docs/concepts/storage/volumes/#portworxvolume) - Portworx volume
-* [`quobyte`](/docs/concepts/storage/volumes/#quobyte) - Quobyte volume
 * [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume
-* [`scaleIO`](/docs/concepts/storage/volumes/#scaleio) - ScaleIO volume
-  (**deprecated**)
-* [`storageos`](/docs/concepts/storage/volumes/#storageos) - StorageOS volume
 * [`vsphereVolume`](/docs/concepts/storage/volumes/#vspherevolume) - vSphere VMDK volume
 
+The following types of PersistentVolume are currently deprecated. The support is still available but will be removed soon.
+
+* [`cinder`](/docs/concepts/storage/volumes/#cinder) - Cinder (OpenStack block storage)
+  (**deprecated** in v1.18, support removal is planned in v1.26)
+* [`flocker`](/docs/concepts/storage/volumes/#flocker) - Flocker storage
+  (**deprecated** in v1.22, support removal is planned in v1.26)
+* [`quobyte`](/docs/concepts/storage/volumes/#quobyte) - Quobyte volume
+  (**deprecated** in v1.22, support removal is planned in v1.26)
+* [`storageos`](/docs/concepts/storage/volumes/#storageos) - StorageOS volume
+  (**deprecated** in v1.22, support removal is planned in v1.26)
+
+The support of following types of PersistentVolume are removed.
+
+* `photonPersistentDisk` - Photon controller persistent disk.
+  (**removed** in v1.16)
+* [`scaleIO`](/docs/concepts/storage/volumes/#scaleio) - ScaleIO volume
+  (**removed** in v1.22)
```

Comment:
I'd write:
```suggestion
Older versions of Kubernetes also supported two other in-tree
PersistentVolume types: `photonPersistentDisk` (not available after v1.15)
and `scaleIO` (not available after v1.21).
```

I suggested the new wording because “are removed” is written more like the present tense, whereas this suggested text is written so it can stay in the document and fit there even many minor releases later.

Reply:
Done.